src/components/vm/vmMigrateDialog.scss

```diff
@@ -6,3 +6,7 @@
 .popover-headline {
     margin-bottom: var(--pf-global--spacer--xs);
 }
+
+#delete-storage {
```

Comment:
I am not fond of custom CSS for nesting form fields. You can have some kind of nesting with the following code:
``` diff
diff --git src/components/vm/vmMigrateDialog.jsx src/components/vm/vmMigrateDialog.jsx
index 2a41678c..d96661b4 100644
--- src/components/vm/vmMigrateDialog.jsx
+++ src/components/vm/vmMigrateDialog.jsx
@@ -110,7 +110,8 @@ const StorageRow = ({ storage, setStorage, temporary, deleteStorage, setDeleteSt
                                <OutlinedQuestionCircleIcon />
                            </button>
                        </Popover>
-                   }>
+                   }
+                   isStack>
             <Radio id="shared"
                    name="storage"
                    label={_("Storage is at a shared location")}
@@ -120,13 +121,15 @@ const StorageRow = ({ storage, setStorage, temporary, deleteStorage, setDeleteSt
                    name="storage"
                    label={_("Copy storage")}
                    isChecked={storage === "copy"}
-                   onChange={() => setStorage("copy")} />
-            {storage === "copy" && !temporary &&
-                <Checkbox id="delete-storage"
-                          isChecked={deleteStorage}
-                          label={_("Delete storage from origin host after migration")}
-                          onChange={value => setDeleteStorage(value)} />
-            }
+                   onChange={() => setStorage("copy")}
+                   body={
+                       (storage === "copy" && !temporary)
+                           ? <Checkbox id="delete-storage"
+                                       isChecked={deleteStorage}
+                                       label={_("Delete storage from origin host after migration")}
+                                       onChange={value => setDeleteStorage(value)} />
+                           : null
+                   } />
         </FormGroup>
     );
 };
diff --git src/components/vm/vmMigrateDialog.scss src/components/vm/vmMigrateDialog.scss
index 9c8a1dc7..28927a75 100644
--- src/components/vm/vmMigrateDialog.scss
+++ src/components/vm/vmMigrateDialog.scss
@@ -6,7 +6,3 @@
 .popover-headline {
     margin-bottom: var(--pf-global--spacer--xs);
 }
-
-#delete-storage {
-    margin-left: var(--pf-global--spacer--xl);
-}

```

Reply:
Thanks, works like charm